### DISCUSSION TITLE
How to make those product demo gifs in readme?

### DISCUSSION THREAD

**Discussion by Metal-joker** [toeverything/AFFiNE#2966] [Q&A] [answered]
like this [demo in readme](https://user-images.githubusercontent.com/79301703/230893796-dc707955-e4e5-4a42-a3c9-18d1ea754f6f.gif)
Is there a tool for this?

**Comment by himself65** [comment]
@ShortCipher5

**Comment by ShortCipher5** [comment, marked as answer]
I believe this particular GIF was put together in Screen Studio: https://www.screen.studio/.
